### DISCUSSION TITLE
nuevo en Python, AYUDAAA

### DISCUSSION THREAD

**Discussion by Moncef1985** [community/community#43155] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

hola  a todos, empeze un curso de Python, y nos dieron un ejercicio que no llegue a entender, si hay alguien que me pueda ayudar.

Gracias

**Comment by LuisContrerasGlz** [comment]
Hola buen dia,

Con gusto te puedo ayudar, solo seria importante explicar o describir de que se trata el ejercicio por este medio así podemos revisar que sucede :-) saludos

**Reply by Moncef1985** [reply]
Muchísimas gracias por tu tiempo.
Son 2 ejercicios 
Este es el 1:

![image](https://user-images.githubusercontent.com/121823980/210364759-6dc6a2bd-8d9c-49fb-ab11-81c95341b4bf.jpeg)

Este es el 2:

![image](https://user-images.githubusercontent.com/121823980/210364826-d921cd37-3f8a-4d6a-a31e-3d6073665abd.jpeg)

**Reply by LuisContrerasGlz** [reply]
1. para el primero seria:

def imc(masa, estatura):
imc = masa/(estatura ** 2)
return imc

imc = imc(masa, estatura)

2. para el segundo seria:

def velocidad(distancia, tiempo):
resultado = ""

# Calculamos la velocidad en km/h

velocidad_km_h = distancia / tiempo * 3600

# Calculamos la velocidad en m/s

velocidad_m_s = distancia / tiempo * 1000

resultado = "La velocidad es de {} km/h o {} m/s".format(velocidad_km_h, velocidad_m_s)

return resultado

Espero te sirva :-) si te sirve podrías marcar mi respuesta?
Saludos

**Reply by Moncef1985** [reply]
Hola de nuevo, el ejercicio Nr. 1 fue correcta, pero el segundo ejercicio me salió error, no se lo q hice mal

![image](https://user-images.githubusercontent.com/121823980/210447274-c295a142-0a75-4b98-88f5-caffb1fc9819.jpeg)


Y tengo otro ejercicio
Nr.3

![image](https://user-images.githubusercontent.com/121823980/210447730-07a04374-ea6d-45ea-803f-38b945568fb2.jpeg)

Gracias por tu ayuda

**Reply by LuisContrerasGlz** [reply]
Hola de nuevo, 

Trata de correr de nuevo el segundo, lo acabo de correr en mi computadora y si funciono jeje, parece que es algo con la plataforma. deberia de funcionar :-) 

Lo de la respuesta me refiero al boton con flecha para arriba que sale aqui :-) Claro si consideras util la respuesta por supuesto

**Comment by jfullstackdev** [comment]
you can post the snippet here, so we can take a look too

**Reply by Moncef1985** [reply]
Thank you for your time!!
So I have a exercises that I don't understand how i hace yo do it 


![image](https://user-images.githubusercontent.com/121823980/210448195-2f4bed36-874a-4f83-8d6a-5a1424155ecf.jpeg)
